src/main/java/frc/robot/Constants.java

```diff
@@ -138,4 +137,14 @@ public static class LedsConstants {
         public static final int LENGTH = 0;
 
     }
+    public static final class ShooterConstants {
+        public static final int LEADER_SHOOTER_MOTOR_ID = 30;
+        public static final int FOLLOWER_SHOOTER_MOTOR_ID = 31;
+        public static final int LEADER_LINEAR_MOTOR_ID = 31;
+        public static final int FOLLOWER_LINEAR_MOTOR_ID = 31;
+        public static final int SHOOTER_BEAMBREAK_CHANNEL = 32;
+
+        public static final Gains PID_SHOOTER_CONSTANT = new Gains(0,0,0);
```

Comment:
No need to to add the "CONSTANT" at the end..

Also write shooter first, and then PID